src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Statuses.cs

```diff
@@ -74,36 +86,43 @@ public class Content
         /// <summary>
         ///  Gets or sets the Id. The numeric message status ID.
         /// </summary>
+        [JsonPropertyName("id")]
         public int Id { get; set; }
 
         /// <summary>
         ///  Gets or sets the LastUpdate. Date and time of status.
         /// </summary>
+        [JsonPropertyName("lastUpdate")]
         public DateTime LastUpdate { get; set; }
 
         /// <summary>
         ///  Gets or sets the Status. The message status. Can be one of: OPPRETTET, SENDT, MOTTATT, LEVERT, LEST, FEIL, ANNET, INNKOMMENDE_MOTTATT, INNKOMMENDE_LEVERT, LEVETID_UTLOPT.
         /// </summary>
+        [JsonPropertyName("status")]
         public string Status { get; set; }
 
         /// <summary>
         ///  Gets or sets the Description
         /// </summary>
+        [JsonPropertyName("description")]
         public string Description { get; set; }
 
         /// <summary>
         ///  Gets or sets the ConvId. The numeric conversation ID.
         /// </summary>
+        [JsonPropertyName("convId")]
         public int ConvId { get; set; }
 
         /// <summary>
         ///  Gets or sets the ConversationId. The numeric conversation ID.
         /// </summary>
+        [JsonPropertyName("conversationId")]
         public string ConversationId { get; set; }
 
         /// <summary>
         ///  Gets or sets the MessageId. The messageId. Typically an UUID.
         /// </summary>
+        [JsonPropertyName("messageId")]
```

Comment:
I was hoping this could be unnecessary if we told the serializer to use CamelCasing. Where did this issue pop up?